cmd/kubeadm/app/cmd/cmd.go

```diff
@@ -93,5 +95,7 @@ func NewKubeadmCommand(f cmdutil.Factory, in io.Reader, out, err io.Writer) *cob
 	experimentalCmd.AddCommand(phases.NewCmdPhase(out))
 	cmds.AddCommand(experimentalCmd)
 
+	utilfeature.DefaultFeatureGate.AddFlag(cmds.PersistentFlags())
```

Comment:
So will this percolate those flags to the components?  It just looks like it adds them to the binary?